src/main/java/com/softserve/academy/spaced/repetition/service/impl/UserServiceImpl.java

```diff
@@ -109,7 +109,7 @@ public User addExistingDeckToUsersFolder(Long userId, Long deckId) {
         userRepository.save(user);
         return user;
     }
-
+    //TODO: Move to separate class
     @Override
     public String getNoAuthenticatedUserEmail() throws NotAuthorisedUserException {
```

Comment:
Do we need this method?
It seems that we can delete it and use `getAuthorizedUser()`, because when we get `User`, we can also get `email` of that user.